tracer/test/test-applications/integrations/Samples.NetActivitySdk/Program.cs

```diff
@@ -0,0 +1,190 @@
+using System.Diagnostics;
+using System.Threading.Tasks;
+using System;
+using System.Collections.Generic;
+
+namespace NetActivitySdk;
+
+public static class Program
+{
+    private static ActivitySource _source;
+
+    public static async Task Main(string[] args)
+    {
+        EnsureAutomaticInstrumentationEnabled();
+
+        _source = new ActivitySource("Samples.NetActivitySdk");
+
+        using (var rootSpan = _source.StartActivity("RootSpan")) // 1 span (total 1)
+        {
+            RunActivityAddTags(); // 1 span (total 2)
+            RunActivitySetTags(); // 1 span (total 3)
+            RunActivityAddEvent(); // 5 spans (total 8)
+            RunActivityAddBaggage(); // 2 spans (total 10)
+        }
+
+        // needs to be outside of the above root span as we don't want a parent here
+        RunActivityConstructors(); // 4 spans (total 14)
+        RunActivityUpdate(); //  9 spans (total 23)
+        await Task.Delay(1000);
```

Comment:
nit: Not required, but it could be helpful to add an ActivityListener that prints diagnostic information about the Activity to the console OnStop so we can compare/contrast the real Activity objects with the emitted Datadog spans. Looking at the snapshots I wasn't sure what information was expected in certain scenarios and this could help with that

Reply:
Good suggestion and that makes sense, working on adding that now